src/main.py

```diff
@@ -61,14 +53,34 @@ def handle_urls(url: str, method: str) -> None:
             check_result = check_ok(url)
         case 'ping':
             hostname = obtain_hostname(url)
-            check_result = my_ping(hostname)
+            check_result = check_ping(hostname)
     
     if not check_result:
-        tg_res = run(make_request_to_telegram(url, method))
-        print(tg_res)
+        handle_retries(url,method,5)
     else:
         print(f'✅ Execution of {method} with address {url} succeeded')
 
+def handle_retries(url: str, method: str, tries: int) -> None:
+
+    tg_res = run(make_request_to_telegram(url, method))
+
+    for mess in tg_res:
+        print(f"mess: {mess}")
+   
+        if not mess['ok']:
```

Comment:
In order to reduce operations, is better to move `tries > 0` in this here

`if tries > 0 and not mess['ok']:`